modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -72,6 +72,9 @@ A batch job contains one or more batch steps that act upon records as they move
 Each batch step in a batch job contains processors that act upon a record to transform, route, enrich, or otherwise process data contained within it. By leveraging the functionality of existing Mule processors, the batch step offers a lot of flexibility regarding how a batch job processes records. See xref:batch-filters-and-batch-aggregator.adoc[Refining Batch Steps Processing] for more information.
 
 A batch job executes when the flow reaches the process-records section of the batch job. When triggered, Mule creates a new batch job instance.
+When the job instance becomes executable, the batch engine submits a task for each record block to the
+I/O pool to process each record. Parallelism occurs automatically, at the record
```

Comment:
Will your audience know what is meant by "parallelism occurs"? (cuz' I don't....)
I don't think of parallelism as being an action, just a result of some other action.

Reply:
"Parallelism is applied automatically..."?

Reply:
They should understand. It's not really "applied." Things are processed in parallel instead of sequentially.